spec/requests/halls_request_spec.rb

```diff
@@ -1,46 +1,48 @@
-require "rails_helper"
+# frozen_string_literal: true
 
-RSpec.describe "Halls requests" do
-  describe "GET /halls" do
-    let!(:hall) { Hall.create(name: "Some name", capacity: 500 ) }
+require 'rails_helper'
 
-    it "works and return status 200" do
-      get("/halls")
+RSpec.describe 'Halls requests' do
+  describe 'GET /halls' do
+    let!(:hall) { Hall.create(name: 'Some name', capacity: 500) }
```

Comment:
It would be great if you use factories(`FactoryBot`) here and in other examples :)